cypress/e2e/eholdings/title+package/add-agreement-to-package.cy.js

```diff
@@ -0,0 +1,115 @@
+import { Permissions } from '../../../support/dictionary';
+import { EHoldingsTitles, EHoldingsTitlesSearch } from '../../../support/fragments/eholdings';
+import { AssignedUsers } from '../../../support/fragments/settings/eholdings';
+import TopMenu from '../../../support/fragments/topMenu';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+import Users from '../../../support/fragments/users/users';
+
+describe('eHoldings', () => {
+  describe('Title+Package', () => {
+    const testData = {
+      agreement: 'Default Agreement',
+      title: 'Fish Biology',
+      titleId: '',
+      titleName: '',
+      titlePackages: [],
+      user: {},
+    };
+
+    before('Create test data', () => {
+      cy.createTempUser([
+        Permissions.uiAgreementsAgreementsEdit.gui,
+        Permissions.uiAgreementsSearchAndView.gui,
+        Permissions.moduleeHoldingsEnabled.gui,
+        Permissions.uiSettingsEHoldingsViewAccessStatusTypes.gui,
+        Permissions.uiSettingsEHoldingsViewCustomLabel.gui,
+        Permissions.uiSettingsEHoldingsViewSettings.gui,
+        Permissions.uiAgreementsSearch.gui,
+      ]).then((userProperties) => {
+        testData.user = userProperties;
+
+        AssignedUsers.assignUserToDefaultCredentialsViaApi({ userId: testData.user.userId });
```

Comment:
Do you need some chanfing regarding this pr?
https://github.com/folio-org/stripes-testing/pull/2759

Reply:
Removed assignement action